src/DesignKit.tsx

```diff
@@ -2987,6 +2988,234 @@ const MessageSection: React.FC = () => {
   );
 };
 
+const ElevationSection: React.FC = () => {
+  const { ui } = useUI();
+  const cssSulfix = useMemo(
+    () => (ui.darkLight === DarkLight.Dark ? 'Dark' : 'Light'),
+    [ui.darkLight],
+  );
+
+  return (
+    <ComponentSection id="Elevation" title="Elevation">
+      <AntDCard>
+        <Paragraph>
+          The Elevation module is a set of CSS variables and classes that serves the purpose of
+          highlighting levels of interaction in the application (like Cards, context menus, etc).
+        </Paragraph>
+      </AntDCard>
+      <AntDCard title="Usage">
+        <Paragraph>
+          To begin with, the Elevation module can be used with the CSS variables only, there are
+          variables for each aspect of the elevation module:
+        </Paragraph>
+        <br />
```

Comment:
fix: All of these sections should be more clearly delineated with an <hr /> or bold titles, ideally both. Look at the Form section for an example

Reply:
Oh lol I didn't realize it would render an `<hr />`